content/2_Bronze/Intro_Sorting.mdx

```diff
@@ -121,22 +146,53 @@ Although you usually do **not** need to know how sorting is implemented, you sho
 
 </LanguageSection>
 
-### (Dynamic) Arrays
+### Static Arrays
 
 <LanguageSection>
 
 <CPPSection>
 
-In order to sort a dynamic array, use `sort(v.begin(), v.end())` (or `sort(begin(v),end(v))`), whereas static arrays require `sort(arr, arr + N)` where $N$ is the number of elements to be sorted. The default sort function sorts the array in ascending order.
+To sort static arrays, use `sort(arr, arr + N)` where $N$ is the number of elements to be sorted. Range can also be specified by replacing `arr` and `arr + N` with the intended range.
```

Comment:
> Range can also be specified

The range?

Specify that `sort(arr+1,arr+4)` sorts `arr[1]` up to (but not including) `arr[4]`